extensions/adobe/experience/customerJourneyManagement/message-experienceevent.schema.json

```diff
@@ -0,0 +1,110 @@
+                  "description": "Email channel fields for the Messageprofle associated with the Adobe CJM Experience Event."
+                }
+              },
+              "required": ["https://ns.adobe.com/xdm/experience/customerJourneyManagement/emailChannelContext"]
+            },
+            {
+              "properties": {
+                "https://ns.adobe.com/xdm/experience/customerJourneyManagement/pushChannelContext": {
+                  "title": "Push Channel Context",
+                  "$ref": "https://ns.adobe.com/experience/customerJourneyManagement/push-channel-context",
+                  "description": "Push channel fields for the MessageProfile associated with the Adobe CJM Experience Event."
+                }
+              },
+              "required": ["https://ns.adobe.com/xdm/experience/customerJourneyManagement/pushChannelContext"]
+            }
+          ]
+        },
+        "https://ns.adobe.com/xdm/experience/customerJourneyManagement/eventStatus": {
+          "title": "Event Status",
+          "description": "Provide the status for the Delivery, Feedback, Tracking Experience Event for the MessageProfile.",
+          "oneOf": [
+            {
+              "properties": {
+                "https://ns.adobe.com/xdm/experience/customerJourneyManagement/messageDeliveryfeedback": {
+                  "title": "Message Delivery Feedback",
+                  "$ref": "https://ns.adobe.com/experience/customerJourneyManagement/message-delivery-feedback",
+                  "description": "Captures Delivey status of a message."
+                }
+              },
+              "required": ["https://ns.adobe.com/xdm/experience/customerJourneyManagement/messageDeliveryfeedback"]
+            },
+            {
+              "properties": {
+                "https://ns.adobe.com/xdm/experience/customerJourneyManagement/messageInteraction": {
+                  "title": "Message Interaction",
+                  "$ref": "https://ns.adobe.com/experience/customerJourneyManagement/message-interaction",
+                  "description": "Captures interaction made by the profile on the message."
+                }
+              },
+              "required": ["https://ns.adobe.com/xdm/experience/customerJourneyManagement/messageInteraction"]
+            }
+          ]
+        }
+      }
+    }
+  },
+  "required": [
+    "https://ns.adobe.com/xdm/experience/customerJourneyManagement/messageExecution",
+    "https://ns.adobe.com/xdm/experience/customerJourneyManagement/messageProfile",
+    "https://ns.adobe.com/xdm/experience/customerJourneyManagement/channelContext",
+    "https://ns.adobe.com/xdm/experience/customerJourneyManagement/eventStatus",
+    "xdm:directMarketing"
+  ],
+  "allOf": [
+    {
+      "$ref": "https://ns.adobe.com/xdm/common/extensible#/definitions/@context"
+    },
+    {
+      "$ref": "https://ns.adobe.com/xdm/context/experienceevent"
```

Comment:
Mixins do not include the class in their allOf definition. The class will be included in the /schemas that is created along with this mixin.

Reply:
So you want this to be a "global" cross tenant schema. This is technically possible, but not 100% supported in AEP. A user will not see this schema in the UI for example. Also not sure what will happen if they click on the schema of the Dataset. 

You can certainly try it though. To do that remove the "meta:intendedToExtend" (this is what makes it a mixin) and set the "meta:abstract" attribute to false.